docs/spelling_wordlist.txt

```diff
@@ -48,7 +52,9 @@ internet
 invalidations
 IPv4
 IPv
+journactl
```

Comment:
As per https://github.com/ClusterHQ/flocker/pull/1924#discussion_r39366317 this shouldn't be in the spelling wordlist.